cassiopeia/src/main/java/edu/asu/diging/gilesecosystem/cassiopeia/web/EditPropertiesController.java

```diff
@@ -36,12 +39,49 @@
     private ISystemMessageHandler messageHandler;
 
     Map<String, String> ocrTypeMap = new HashMap<>();
+    Map<String, String> langTypeMap = new HashMap<>();
+    String defaultLang = new String();
+
 
     @InitBinder
     protected void initBinder(HttpServletRequest request, ServletRequestDataBinder binder, WebDataBinder validateBinder) {
         validateBinder.addValidators(new SystemConfigValidator());
         ocrTypeMap.put(Properties.OCR_PLAINTEXT, propertyManager.getProperty(Properties.OCR_PLAINTEXT));
         ocrTypeMap.put(Properties.OCR_HOCR, propertyManager.getProperty(Properties.OCR_HOCR));
+        
+        String[] langs = getTessLangs();
+        for(int i=1;i<langs.length;i++) {
+        	langTypeMap.put(langs[i],langs[i]);
+        } 
+        if(langTypeMap.containsKey(Properties.ENGLISH)) {
+        	defaultLang = langTypeMap.get(Properties.ENGLISH);
+        }
+    }
+    
+    public String[] getTessLangs() {
+        String command = "/usr/local/bin/tesseract --list-langs";
```

Comment:
depending on which system we're running on, this path could be anything.

Reply:
Changed the logic; now taking the path value from Properties.